maven-plugin/src/main/java/org/openmrs/maven/plugins/CreateProject.java

```diff
@@ -255,14 +262,26 @@ private void createModule() throws MojoExecutionException {
 		moduleDescription = wizard
 				.promptForValueIfMissingWithDefault(DESCRIPTION_PROMPT_TMPL, moduleDescription, "", "no description");
 
-		wizard.showMessage(MAVEN_INFO);
-		groupId = wizard.promptForValueIfMissingWithDefault(GROUP_ID_PROMPT_TMPL, groupId, "groupId", "org.openmrs.module");
-		while (!groupId.matches("[a-z][a-z0-9.]*")) {
-			wizard.showError("The specified groupId " + groupId
-					+ " is not valid. It must start from a letter and can contain only alphanumerics and dots.");
-			groupId = null;
-			groupId = wizard
-					.promptForValueIfMissingWithDefault(GROUP_ID_PROMPT_TMPL, groupId, "groupId", "org.openmrs.module");
+		if (TYPE_CONTENT_PACKAGE.equals(type)) {
+			wizard.showMessage(CONTENT_MAVEN_INFO);
+			groupId = wizard.promptForValueIfMissingWithDefault(GROUP_ID_PROMPT_TMPL, groupId, "groupId", "org.openmrs.content");
+			while (!groupId.matches("[a-z][a-z0-9.]*")) {
+				wizard.showError("The specified groupId " + groupId
+						+ " is not valid. It must start from a letter and can contain only alphanumerics and dots.");
+				groupId = null;
+				groupId = wizard
+						.promptForValueIfMissingWithDefault(GROUP_ID_PROMPT_TMPL, groupId, "groupId", "org.openmrs.content");
+			}
+		} else {
+			wizard.showMessage(MAVEN_INFO);
+			groupId = wizard.promptForValueIfMissingWithDefault(GROUP_ID_PROMPT_TMPL, groupId, "groupId", "org.openmrs.module");
+			while (!groupId.matches("[a-z][a-z0-9.]*")) {
+				wizard.showError("The specified groupId " + groupId
+						+ " is not valid. It must start from a letter and can contain only alphanumerics and dots.");
+				groupId = null;
+				groupId = wizard
+						.promptForValueIfMissingWithDefault(GROUP_ID_PROMPT_TMPL, groupId, "groupId", "org.openmrs.module");
+			}
```

Comment:
Can't this be simplified?

```suggestion
                String defaultGroupId;
		if (TYPE_CONTENT_PACKAGE.equals(type)) {
		        defaultGroupId = "org.openmrs.content";
			wizard.showMessage(CONTENT_MAVEN_INFO);
	        } else {
	                defaultGroupId = "org.openmrs.module";
			wizard.showMessage(CONTENT_MAVEN_INFO);
		}
		
		// code can be shared here
```